add downcase back in

jacobthill opened the pull request:
Somehow I deleted the downcase

jcoyne commented on sakip_sabanci_config.rb:
```diff
@@ -27,10 +27,10 @@
 to_field 'cho_dc_rights', extract_oai('dc:rights'), strip
 to_field 'cho_edm_type', extract_oai('dc:type'),
-         strip, translation_map('not_found', 'types')
+         transform(strip: true, downcase: true), translation_map('not_found', 'types')
```
This should be 
```
strip, transform(&:downcase), translation_map('not_found', 'types')
```

jcoyne commented on sakip_sabanci_config.rb:
```diff
@@ -27,10 +27,10 @@
 to_field 'cho_format', extract_oai('dc:format'), strip
 to_field 'cho_language', extract_oai('dc:language'), split(';'),
-         strip, translation_map('not_found', 'languages', 'marc_languages')
+         transform(strip: true, downcase: true), translation_map('not_found', 'languages', 'marc_languages')
```
same here.

jacobthill replied:
that doesn't seem to work. Its not downcasing the values as far as I can tell. Any idea what's going wrong?

jcoyne replied:
Ah! We need this merged first: https://github.com/sul-dlss/traject_plus/pull/29 @justinlittman can you take a look?

justinlittman replied:
Done.

jcoyne replied:
Okay, next step would be to cut a 1.1.0 release of traject_plus and then update dlme-transform.

jcoyne replied:
@jacobthill those steps have been taken.  Can you pull the latest dlme-transform code, rebuild the container and see if this would work now?

jacobthill replied:
Yep, I’m on site this week so I’ll do it when I get to campus

jacobthill commented:
It does work now, but the way I was using split was causing problems. I tried to follow the same pattern as we do for downcase e.g. `transform(&:split('.')` but that resulted in a different error. Is there an another way I should be chaining together transformations other than:
`to_field 'cho_contributor', extract_oai('dc:contributor'),
         strip, split('.')`

jcoyne commented:
@jacobthill you don't want to pass `split` to transform. you should use `split('.')`
